tests/nsrt_learning/strips_learning/test_gen_to_spec_learner.py

```diff
@@ -805,7 +805,7 @@ def test_keep_effect_data_partitioning():
     Preconditions: [MachineConfigurableWhileOff(?x0:machine_type)]
```

Comment:
it's nice to see a number of these operators getting simpler, and planning still working!